apache/yunikorn-k8shim
# [YUNIKORN-1775] add namespaces permission to admission cluster role

### What is this PR for?

Admission cluster role doesn't have enough permission to run admission.

### What type of PR is it?

### Todos

### What is the Jira issue?

https://issues.apache.org/jira/browse/YUNIKORN-1775

### How should this be tested?

Using a new cluster to deploy the admission controller. It should not have any error.

```
kubectl create namespace yunikorn
kubectl apply -f deployments/scheduler/admission-controller-rbac.yaml --namespace yunikorn
kubectl apply -f deployments/scheduler/admission-controller-secrets.yaml --namespace yunikorn
kubectl apply -f deployments/scheduler/admission-controller.yaml --namespace yunikorn
```

### Screenshots (if appropriate)

### Questions:
